third_party/BUILD

```diff
@@ -1,4 +1,4 @@
-load("@mypy_integration_pip_deps//:requirements.bzl", "requirement")
+load("@pypi//:requirements.bzl", "requirement")
```

Comment:
I think changing the name of the pip deps repository to `pypi` will break other repositories that set the same for their internal deps. Could you undo this, please?

Reply:
yeah, I agree